desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java

```diff
@@ -173,7 +175,11 @@ protected JComponent getDialogContent() {
             checkBoxPanel.add(checkBox);
         }
 
-        WidgetUtils.addToGridBag(checkBoxPanel, formPanel, 0, 1, 2, 1);
+        JScrollPane checkBoxScrollPane = WidgetUtils.scrolleable(checkBoxPanel);
+        checkBoxScrollPane.setPreferredSize(new Dimension(checkBoxPanel.getWidth(), 300));
+        checkBoxScrollPane.setVerticalScrollBarPolicy(JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED);
+        checkBoxScrollPane.setBackground(_outerPanel.getBackground());
```

Comment:
Is this really needed? Normally a scrollpane does not fill the content, it only adds the scroll bars on top.